components/TextFields/src/MDCTextInputBorderView.h

```diff
@@ -0,0 +1,48 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+@interface MDCTextInputBorderView : UIView <NSCopying>
+
+/**
+ The color of the area inside the border.
+
+ Default is clear.
+ */
+@property(nonatomic, nullable, strong) UIColor *borderFillColor UI_APPEARANCE_SELECTOR;
```

Comment:
nullable or null_resettable?
If null_resettable, happy to open an issue and change in a later PR.

Reply:
All UI_APPEARANCE_SELECTOR properties have to be nullable :(